pkg/binary/envoy/controlplane/testdata/cert-chain.pem

```diff
@@ -1,21 +0,0 @@
------BEGIN CERTIFICATE-----
```

Comment:
these files were dead (never read). when we get to doing cert tests, we can add relevant ones but don't need to store dead files meanwhile